fwprovider/provider.go

```diff
@@ -123,6 +115,26 @@ func (p *proxmoxProvider) Schema(_ context.Context, _ provider.SchemaRequest, re
 					"Supported values: `1.0|1.1|1.2|1.3`. Defaults to `1.3`.",
 				Optional: true,
 			},
+			"auth_payload": schema.StringAttribute{
+				Description: "The pre-authd full Ticket Payload json for the Proxmox VE API (takes precedence over auth_ticket).",
+				Optional:    true,
+				Sensitive:   true,
+			},
+			"auth_ticket": schema.StringAttribute{
+				Description: "The pre-authd Ticket for the Proxmox VE API.",
+				Optional:    true,
+				Sensitive:   true,
+			},
+			"csrf_prevention_token": schema.StringAttribute{
+				Description: "The pre-authd CSRF Prevention Token for the Proxmox VE API.",
+				Optional:    true,
+				Sensitive:   true,
+			},
+			"api_token": schema.StringAttribute{
+				Description: "The API token for the Proxmox VE API.",
+				Optional:    true,
+				Sensitive:   true,
+			},
```

Comment:
Could you pls. sort all the attributes alphabetically? That wasn't a strict rule in the legacy code, but I'd like to follow it in the new PRs

Reply:
ahh cool, yes will get to that (i was wondering about that; i did a sort (of the auth-stuff) by precedence ) - how about the function call where i re-ordered the inputs - leave it or sort by alpha?
( eg it is now:  `func NewCredentials(username, password, otp, apiToken string, authTicket string, csrfPreventionToken string, authPayload string) (*Credentials, error)` )